tests/resources/telescope_positions-North-utm.ecsv

```diff
@@ -5,8 +5,8 @@
 # - {name: sequence_number, datatype: string, description: sequence number}
 # - {name: utm_east, unit: m, datatype: float32}
 # - {name: utm_north, unit: m, datatype: float32}
-# - {name: alt, unit: m, datatype: float32, description: altitude of the array element}
-# - {name: geo_code, datatype: string, description: geographical position code}
+# - {name: alt, unit: m, datatype: float32, description: telescope position upwards relative to NNN}
```

Comment:
"upwards relative to NNN"? The looks like a placeholder.

Reply:
Typo, was supposed to be 'NN' - I replaced it by Normal Null to make it clearer.

Reply:
Is Normal Null clearer? From Googling it seems to be an archaic German convention. Is it not `altitude above sea level` rather than `upwards relative to Normal Null`?

Reply:
I don't think this is archaic! I learned it in school.

Atitude is a very complex thing, but let's not spend so much time on it at this point and simply write 'altitude above sea level'.

Reply:
> I don't think this is archaic! I learned it in school.

You should edit Wikipedia then =)